script/bulk_import/generic_bulk.rb

```diff
@@ -1953,11 +2023,21 @@ def import_tags
         )
       @tag_mapping[row["id"]] = tag.id
 
-      if row["tag_group_id"]
-        TagGroupMembership.find_or_create_by!(
-          tag_id: tag.id,
-          tag_group_id: @tag_group_mapping[row["tag_group_id"]],
-        )
+      if row["tag_group_id"] && !row["tag_group_id"].empty?
+        intermediate_group_ids = JSON.parse(row["tag_group_id"])
+
+        intermediate_group_ids.each do |intermediate_group_id|
+          discourse_tag_group_id = @tag_group_mapping[intermediate_group_id]
+
+          if discourse_tag_group_id
+            TagGroupMembership.find_or_create_by!(
+              tag_id: tag.id,
+              tag_group_id: discourse_tag_group_id,
+            )
+          else
+            puts "Warning: Intermediate tag group ID #{intermediate_group_id} from row not found in @tag_group_mapping for tag '#{tag.name}'"
+          end
+        end
```

Comment:
Did you update `tag_group_id` in the converter to a JSON column? 

You’d typically expect `tag_group_id` to be an integer. If we want to support multiple tag group IDs, it might be better to add a separate `tag_group_ids` column and leave `tag_group_id` as is since we don’t have time to update all converters right now. We can just handle both columns here for now